lib/ansible/modules/remote_management/redfish/redfish_facts.py

```diff
@@ -301,6 +315,8 @@ def main():
             for command in command_list:
                 if command == "GetManagerNicInventory":
                     result["manager_nics"] = rf_utils.get_multi_nic_inventory(category)
+                elif command == "GetVirtualMedia":
+                    result["virtual_media"] = rf_utils.get_multi_virtualmedia(category)
```

Comment:
Since GetVirtualMedia is only a command in the "Manager" category (and not in the "Systems" category), you do not need to pass in a `category` param here. See also my previous get_multi_virtualmedia() comment in redfish_utils.py.

Reply:
Thanks for your detailed explanation, billdodd. I will remove category parameter. Sine aggregate() is only for systems, I plan to keep get_multi_virtualmedia(), but I will remove all systems related logic. And I also found 1 issue in _find_managers_resource(), which can only keep last Manager uri in self.manager_uri, not one list. because manager_uri is being used by multiple positions, I plan to fix it in another PR.

Reply:
@renxulei Yes, sorry about that. I missed that this was for Managers rather that Systems. Your update looks good. When you do the future PR to improve the handling of multiple managers, it may be a good idea the rename the aggregate() method to aggregate_systems() and create a new aggregate_managers() method.

Reply:
@billdodd Thanks for your suggestion, I will fix the code as you suggested in future PR.